deployment/k8sPaiLibrary/maintainlib/kubectl_install.py

```diff
@@ -57,21 +57,21 @@ def kubectl_install(self):
 
 
     def kubectl_configuration_generate(self):
-
+        com = self.cluster_config
         self.logger.info("Generate the configuation file of kubectl.")
 
-        if self.cluster_config != None:
+        if com != None:
```

Comment:
I'm not sure why we need the assignment?
Could we use cluster_config directly?

Reply:
No，I will left this to make it as another approch